mods/ts/rules/world.yaml

```diff
@@ -54,6 +54,7 @@
 		PipColor: Red
 		AllowedTerrainTypes: Clear
 		AllowUnderActors: true
+		AllowUnderBuildings: true
```

Comment:
although note that this will also let them grow under walls, which we don't want.  It might be better to leave this disabled for now, and then explicitly enable it once we can support the behaviour and building damage properly.